<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCash - Desktop</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>195</width>
     <height>347</height>
    </rect>
   </property>
   <property name="title">
    <string>Ordem</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Papel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePapel"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quantidade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinQuantidade"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Preço:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="precoOrdem">
        <property name="prefix">
         <string>R$</string>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Corretora:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboCorretoraOrdem"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Operação</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioCompra">
         <property name="text">
          <string>Compra</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioVenda">
         <property name="text">
          <string>Venda</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="InsereOrdem">
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dataOrdem"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>195</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Proventos</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Papel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePapelProvento"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Valor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="valorProvento">
        <property name="prefix">
         <string>R$</string>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTipoProvento"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Corretora:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboCorretoraProventos"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="insereProvento">
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dataProvento"/>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Carteira</string>
    </attribute>
    <widget class="QTableView" name="carteiraTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Operações</string>
    </attribute>
    <widget class="QTableView" name="operacaoTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Proventos</string>
    </attribute>
    <widget class="QTableView" name="proventoTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>401</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
